--- a/etl/02_etl/cdm_cdm_source.sql
+++ b/etl/02_etl/cdm_cdm_source.sql
@@ -54,7 +54,7 @@
     'MIT'                                   AS cdm_holder,
     'https://github.com/OHDSI/MIMIC/'       AS cdm_etl_reference,
     to_date('2020-09-01', 'YYYY-MM-DD')    AS source_release_date, -- to look up
-    CURRENT_DATE()                          AS cdm_release_date,
+    CURRENT_DATE                            AS cdm_release_date,
     '5.3.1'                                 AS cdm_version,
     v.vocabulary_version                    AS vocabulary_version,
